specification/2.0/schema/texture.schema.json

```diff
@@ -5,55 +5,18 @@
     "description": "A texture and its sampler.",
     "allOf": [ { "$ref": "glTFChildOfRootProperty.schema.json" } ],
     "properties": {
-        "format": {
-            "type": "integer",
-            "description": "The texture's format.",
-            "enum": [ 6406, 6407, 6408, 6409, 6410 ],
-            "gltf_enumNames": [ "ALPHA", "RGB", "RGBA", "LUMINANCE", "LUMINANCE_ALPHA" ],
-            "default": 6408,
-            "gltf_detailedDescription": "The texture's format.  Valid values correspond to WebGL enums: `6406` (ALPHA), `6407` (RGB), `6408` (RGBA), `6409` (LUMINANCE), and `6410` (LUMINANCE_ALPHA).",
-            "gltf_webgl": "`texImage2D()` format parameter"
-        },
-        "internalFormat": {
-            "type": "integer",
-            "description": "The texture's internal format.",
-            "enum": [ 6406, 6407, 6408, 6409, 6410 ],
-            "gltf_enumNames": [ "ALPHA", "RGB", "RGBA", "LUMINANCE", "LUMINANCE_ALPHA" ],
-            "default": 6408,
-            "gltf_detailedDescription": "The texture's internal format.  Valid values correspond to WebGL enums: `6406` (ALPHA), `6407` (RGB), `6408` (RGBA), `6409` (LUMINANCE), and `6410` (LUMINANCE_ALPHA).  Defaults to same value as format.",
-            "gltf_webgl": "`texImage2D()` internalFormat parameter"
-        },
         "sampler": {
             "allOf": [ { "$ref": "glTFid.schema.json" } ],
-            "description": "The index of the sampler used by this texture."
+            "description": "The index of the sampler used by this texture. When undefined, a sampler with repeat wrapping and auto filtering should be used."
```

Comment:
What exactly does "auto filtering" mean? Why were the default min/mag filter values removed from the spec? I'm writing an importer.